src/SFA.DAS.LearnerDataMismatches.Web/Pages/DataLockHelpCentreLink.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections.Generic;
+
+namespace SFA.DAS.LearnerDataMismatches.Web.Pages
+{
+    public class DataLockHelpCentreLink
+    {
+        public string Name { get; set; }
+        public string Description { get; set; }
+        public string Url { get; set; }
+
+        public static DataLockHelpCentreLink Create(Domain.DataLock @lock)
+        {
+            links.TryGetValue(@lock, out var link);
+
+            return new DataLockHelpCentreLink
+            {
+                Name = @lock.ToString(),
+                Description = link.description,
+                Url = MistmatchArticleUrl,
+            };
+        }
+
+        private const string MistmatchArticleUrl = "https://help.apprenticeships.education.gov.uk/hc/en-gb/articles/360008448299-Data-mismatch-errors";
+
+        private static readonly Dictionary<Domain.DataLock, (string url, string description)> links
+            = new Dictionary<Domain.DataLock, (string url, string description)>
+            {
+                {Domain.DataLock.Dlock01, ("", "No matching UKPRN record found") },
```

Comment:
`url` part of the Tuple is not in use, I am assuming that each datalock will have specific URL which will be determined later? perhaps use string.Empty

Reply:
I have removed the unnused URL item.